modules/ROOT/pages/_partials/link-anypoint-platform-accounts.adoc

```diff
@@ -0,0 +1,94 @@
+// tag::profileLinkingContextForUsers[]
+[[profile-linking-context-users]]
+= Linking Your Anypoint Platform Profiles
+
+When you log in to an Anypoint Platform using an email address that is associated with more than one profile in the same organization, you are prompted to link your profiles together. Your options for linking profiles vary based on whether the profiles use the Anypoint identity provider (IdP) or an external IdP that uses single sign-on (SSO). Anypoint Platform supports:
+
+* Linking an SSO profile to a local profile (login managed by Anypoint IdP)
+* Linking an SSO profile to another SSO profile that uses a different IdP to log in
+* Linking an SSO profile to multiple other profiles that each use different IdPs to log in
+
+Anypoint Platform does not support linking multiple profiles that use the same IdP. You can link up to 25 user profiles.
```

Comment:
Another limitation is that we're supporting Openid providers at this phase, which means all the above Idps are openid Idps.

Reply:
```suggestion
Anypoint Platform does not support linking multiple profiles that use the same IdP. The IdP must be an OpenID Connect (OIDC) provider. You can link up to 25 user profiles.
```